.github/pull_request_template.md

```diff
@@ -0,0 +1,10 @@
+related tracking issue:
```

Comment:
I'm tempted to say that we should do without this line and just include the comment that might be sufficient to remind people?

```suggestion

```